src/router.tsx

```diff
@@ -10,6 +11,7 @@ const Routes = () => {
       <Switch>
         <Route exact path="/" component={HomeComponent} />
         <Route exact path="/questions" component={QuestionContainer} />
+        <Route exact path="/meldinger" component={Chat} />
```

Comment:
Vi må finne ut om vi skal ha norske eller engelse urler. Jeg stemmer kanskje for engelske slipper vi noe styr med æ,ø,å

Reply:
Kan gå med på det!